hw3/src/main/kotlin/Main.kt

```diff
@@ -0,0 +1,14 @@
+fun getLength(str: String?) : Int{
+
+    if (str != null) {
+        println("String of length is null")
+        else run {
+            println("Empty string")
+        }
+        return str.length
+    };
```

Comment:
You have an error here, Make sure you put the else statement in the right place, And there is no need to use 'run'